packages/grpc-js/src/subchannel.ts

```diff
@@ -121,7 +121,7 @@ export class Subchannel {
   /**
    * Timer reference tracking when the most recent ping will be considered lost
    */
-  private keepaliveTimeoutId: NodeJS.Timer;
+  private keepaliveTimeoutId?: NodeJS.Timer;
```

Comment:
This field should be possibly `null` instead of optional. See for example `session` and `channelzSocketRef` in this class.